spherecluster/von_mises_fisher_mixture.py

```diff
@@ -110,9 +132,9 @@ def _S(kappa, alpha, beta):
     See "S <-" in movMF.R and utility function implementation notes from
     https://cran.r-project.org/web/packages/movMF/index.html
     """
-    kappa = 1.0 * np.abs(kappa)
-    alpha = 1.0 * alpha
-    beta = 1.0 * np.abs(beta)
+    kappa = 1. * np.abs(kappa)
```

Comment:
Curious about the change in style for each of these floats?  

Since this PR is mostly focused on adding the log normalization factor, can we also change these back so the diff can reflect just the salient changes.

Reply:
I've noticed these changes when I `git diff`ed, and I am not sure how they happened, I definitely don't remember making such changes.. besides being unrelated to the log normalization, I don't change style in other peoples code.

Reply:
Once I pulled the changes from your latest develop branch, these style changes were back to include the ".0" in floats. I also changed it in the new log-scale method.

Reply:
sorry I probably messed up my local fork somehow. let me check and I'll update soon

Reply:
See my check list in the main PR comment area,  if you run `black` on the file when you are done, it should take care of everything for you.  The final diffs for this file should be pretty minimal (just the new function, imports, and change in call site).

Reply:
Yes I already fixed all style issues, but adding a test method will take some time (I'm pretty busy as the moment for side-work). So in a few days I'll add the test method.
Btw in my fork, does it matter if I add the changes to `develop` branch or another (mine) branch? for the PR that is

Reply:
I don't think the branch matters (more of a decision on your end) since they are separate develop branches.  Your branch can be named anything  like `feature/normalize_log_scale` or leaving as develop is fine, if thats simpler for you.

However,  I don't see the changes reflect in the PR so you'll need to figure out how to pull it into your existing develop branch or do a new PR.